profiles/kentik_snmp/cisco/cisco-wlc.yml

```diff
@@ -163,6 +163,19 @@ metrics:
           coldStandbyRedundant: 6
           warmStandbyRedundant: 7
           hotStandbyRedundant: 8
+    
+  # HA Sync Status for AirOS WLCs
+  - MIB: CISCO-LWAPP-HA-MIB
+    symbols:
+      - name: cLHaAvgPeerReachLatency # This object represents average value of peer controller reachable latency in micro seconds
+        OID: 1.3.6.1.4.1.9.9.843.1.5.1
+        poll_time_sec: 60
+      - name: cLHaBulkSyncStatus #  This object represents current status of bulk sync
+        OID: 1.3.6.1.4.1.9.9.843.1.5.3
+        poll_time_sec: 60
+        enum:
+          completed: 1
+          pending: 0
```

Comment:
@aditdeshpande10 - need to validate this one. the only MIB reference I can find shows this as a `SnmpAdminString` data type, which is essentially a plain string and not an integer requiring enumeration. 

If this is indeed a string, you need to replace the `enum` on lines 176-178 with this conversion which will force a metric value of 1 and automatically collect the string as a decorated attribute.:

```yaml
conversion: to_one
```

e.g.;
metricName = 'kentik.snmp.cLHaBulkSyncStatus' will always = 1
but there will be an attribute on that metric named `cLHaBulkSyncStatus` which will hold the actual string.

```sql
FROM Metric SELECT
  latest(cLHaBulkSyncStatus)
WHERE metricName = 'kentik.snmp.cLHaBulkSyncStatus'
```

Reply:
you're not going to be able to collect a string as a dimensional metric, it has to use the `conversion: to_one` or it simply will not work with the Metric API.

re: alerting, you definitely do not need the dimensional metric to change for a proper alert. You can simply count the number of incoming samples per device that don't match your desired state and then alert when the count is more than 1. example:

```sql
FROM Metric SELECT count(*)
FACET entity.name
WHERE cLHaBulkSyncStatus != 'Complete'
```

If this PR is to ask a question, I can close it out. (That's what [Issues](https://github.com/kentik/snmp-profiles/issues) are for)

Reply:
Thanks Zach, I'll try this.